docs/model_card.rst

```diff
@@ -0,0 +1,46 @@
+.. _model_cards:
+
+Model Cards for scikit-learn
+============================
+
+This library allows you to automatically create documentation for models, also
+known as model cards. The model cards consist of two parts, a metadata part for
+model discoverability and markdown part for information related to model. The
+card itself is named as ``README.md``.
```

Comment:
I agree. I think this document should make it clear right at the start that we're talking about the specific way that HF expects model cards to be. Users may come here with their own concepts and then be confused otherwise.